docs/commit-guide.md

```diff
@@ -0,0 +1,131 @@
+# Commit Messages Guide
+
+This guide describes how we adding human and machine readable meaning to commit messages.
```

Comment:
nit: This guide describes how we add human and machine-readable commit messages.